modules/installing-ocp-agent.adoc

```diff
@@ -147,6 +147,30 @@ agent.iso  auth
 $ openshift-install agent create image
 ----
 
+. Optional:  To know  when the bootstrap node (** Node 0 **) reboots, run the following command:
```

Comment:
* "To know" doesn't sound right. I'd suggest something like "To determine" or another option instead.
* Usually bold is reserved for GUI items, so "Node 0" should likely use some other markup, if any at all. There also aren't that many other references to "node 0" in the docs, so I wonder if that's needed at all. The ones I do find have it lowercased too, just FYI.